client/me/developer/main.jsx

```diff
@@ -52,43 +52,20 @@ class Developer extends Component {
 	};
 
 	showDevSurveyNotice = () => {
-		const noticeMessage = this.props.translate(
-			"Hey developer! How do {{i}}you{{/i}} use WordPress.com? Spare a moment? We'd love to hear what you think in a {{surveyLink}}quick survey{{/surveyLink}}.",
-			{
-				components: {
-					i: <i />,
-					surveyLink: (
-						<a
-							href={ this.getSurveyHref() }
-							target="_blank"
-							rel="noopener noreferrer"
-							onClick={ () => {
-								recordTracksEvent( 'calypso_me_developer_survey_clicked' );
-
-								this.props.removeNotice( this.developerSurveyNoticeId );
-							} }
-						/>
-					),
-				},
-			}
-		);
-
-		const noticeProps = {
-			id: this.developerSurveyNoticeId,
-			isPersistent: true,
-			onDismissClick: () => {
-				recordTracksEvent( 'calypso_me_developer_survey_dismissed' );
+		if ( ! this.surveyNoticeWrapper ) {
+			this.surveyNoticeWrapper = document.createElement( 'div' );
+			document.body.appendChild( this.surveyNoticeWrapper );
+		}
 
-				this.setDeveloperSurveyCookie( 'dismissed', 24 * 60 * 60 ); // 1 day
+		this.setState( { isVisibleDevSurveyNotice: true } );
```

Comment:
```suggestion
		this.setState( { isDevSurveyNoticeVisible: true } );
```

Nit. I would suggest renaming the variable so it's more straight forward to read, like regular english.

Reply:
Related to a previous comment, could we use `Developer` instead of just `Dev` here and in other places? Reusing the same form would also be more consistent with `setDeveloperSurveyCookie()`.